src/api/app/assets/javascripts/webui/application/kiwi_editor.js

```diff
@@ -58,12 +58,42 @@ function closeDialog() {
       return false;
     }
   }
-  
+
+  addDefault(dialog);
+
   fields.find(".ui-state-error").addClass('hidden');
   $('.overlay').hide();
   dialog.addClass('hidden');
 }
 
+function revertDialog() {
+  var fields = $(this).parents('.nested-fields');
+  var dialog = fields.find('.dialog');
+
+  $.each(dialog.find('input:visible, select:visible'), function(index, input) {
+    if (input.type == 'checkbox') {
```

Comment:
Let's use === and !== instead. It's more strict (comparing object type and it's value) and reflects better what we expect here (a string with value 'checkbox').

[This article](https://bytearcher.com/articles/equality-comparison-operator-javascript/) explains it nicely